src/ProjectNeon/Assets/Data/Templates/PlayedCard.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayedCard : ScriptableObject
```

Comment:
This one should not be a `ScriptableObject`, since we want to instantiate it from Code.